### PR TITLE
feat/inj name resolution

### PR DESCRIPTION
**Pull request by bangjelkoski**

### CODE REVIEW

File: .gitbook/contracts/injective-name-service.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- Get resolver address
+
+```ts
+import { getNetworkEndpoints, Network } from '@injectivelabs/networks'
+import { ChainId } from '@injectivelabs/ts-types'
+import {
+  ChainGrpcWasmApi,
+  QueryResolverAddress,
+  InjNameServiceQueryTransformer
+  INJ_NAME_REGISTRY_CONTRACT_BY_CHAIN_ID,
```
Comment (kevin-222) [inline .gitbook/contracts/injective-name-service.md:20]: should be INJ_NAME_REGISTRY_CONTRACT_BY_NETWORK

File: .gitbook/contracts/injective-name-service.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+const name = 'allen.inj'
+const resolverAddress = '' /** from above query */
+
+const normalizedName = normalizeName(name)
+const node = nameToNode(normalized)
+
+const query = new QueryInjectiveAddress({ node }).toPayload()
+
+const response = await chainGrpcWasmApi.fetchSmartContractState(
+  registryContractAddress,
```
Comment (kevin-222) [inline .gitbook/contracts/injective-name-service.md:73]: should be `resolverAddress`

File: .gitbook/contracts/injective-name-service.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+)
+
+const name = InjNameServiceQueryTransformer.injectiveNameResponseToInjectiveName(response)
+
+/**
+ * checks can be performed to
+ * validate the address here based on the name
+ * i.e query the address based on the name returned
+ * and compare the one above with the result
+*/
```
Comment (kevin-222) [inline .gitbook/contracts/injective-name-service.md:121]: Verification is required. Because the address of name might be different

Comment (bangjelkoski) [reply .gitbook/contracts/injective-name-service.md:121]: Feel free to do the commit on the PR!

**Pull request merged**